Cargo.lock

```diff
@@ -716,15 +717,15 @@ checksum = "9e1b586273c5702936fe7b7d6896644d8be71e6314cfe09d3167c95f712589e8"
 
 [[package]]
 name = "base64"
-version = "0.21.3"
+version = "0.21.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "414dcefbc63d77c526a76b3afcf6fbb9b5e2791c19c3aa2297733208750c6e53"
+checksum = "35636a1494ede3b646cc98f74f8e62c773a38a659ebc777a2cf26b9b74171df9"
 
 [[package]]
 name = "bcs"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
It looks like you run `cargo update`?

Reply:
For the fix commit yes, and also probably in the build after updating `test-case`

Reply:
For the final version I've removed the `test-case` update from here, preferring to move it to https://github.com/linera-io/linera-protocol/pull/1178 with others of its kind.  In order to add `test-case` to `linera-service` without doing a full `cargo update`, I manually edited `Cargo.lock` to add only the dependency we need to build without updating `Cargo.lock`.